src/Components/MainPage/Header/header.css

```diff
@@ -0,0 +1,16 @@
+.header-container {
+  display: flex;
+  align-content: center;
+  padding: 20px;
+  max-width: 1250px;
+  width: 100%;
+  margin: 0 auto;
```

Comment:
This is how I would do it to make it responsive:

  **display: flex;**
  **align-content: center;** // flexing here won't do anything, but if we'd add some components into header (logo for example) then they will still be aligned well. 
  **padding: 20px;**
  **width: 70vw;** //or change however you see fit. basically here I set the width to the header. It is responsive now, as it's basically 70% of the screen. 70 viewport width.
  **min-width: 600px;** //also quite random // when I drag my screen really narrow, then the header will stop shrinking at some point (makes sense, as you have elements on the page that can't shrink infinitely)
  **max-width: 1250px;** //don't want to spread the page over screen too wide (we would still need to keep the vertical structure) (70% of an ultrawide screen is wiiiiiide, so we limit with 1250px)
  **margin: 0 auto;** // was responsive already, great!!

Just play around with them in the browser inspect and compare the properties, remove something and change values etc. So you'll grasp how they differ. If not, call me and I'll show you :)

Reply:
Love it, I always forget/don't even think of using vw as a measurement!

Reply:
Also for the movielist and trendlist would it be better to leave out a min-width?
After playing around with it it seemed like it would be better if it did shrink down to even as small as showing one movie at a time since it is scrollable, if that makes sense?

Reply:
Yeah, actually, if it all looks good with a really narrow viewport, then why not!

Reply:
Yay, will keep it then!